### DISCUSSION TITLE
Very slow project cards view

### DISCUSSION THREAD

**Discussion by sirex** [community/community#50088] [Projects and Issues]
### Select Topic Area

Bug

### Body

I have a project board view with many cards in it. Most cards are excluded using filters. But board view is very slow it is a bit faster on Chrome and much slower on Firefox. But in both cases, it is too slow.

Any click on board or thinks like moving cards to different columns takes up to half minute to complete. Firefox constantly is showing a bar asking if I want to stop execution of JavaScript on this page, because it take too long.

The boar, that I'm talking about is here:

https://github.com/orgs/atviriduomenys/projects/3/views/1

I tink, there is a bug, that causes project boards to be so slow.

By the way, this is not an issue with small boards with few cards in it.

**Comment by Anmol-Baranwal** [comment]
Slow performance on GitHub project boards can be caused by a variety of factors, including the size of the board, the number of filters applied, and the complexity of the queries being executed by GitHub's servers. Here are a few steps you can take to try to improve performance:

1. Reduce the number of cards displayed on the board: One way to improve performance is to reduce the number of cards displayed on the board. You can do this by using more specific filters or by splitting the board into multiple smaller boards.

2. Use more specific filters: The filters you use can have a big impact on performance. Try to use more specific filters that narrow down the number of cards displayed on the board. For example, you could filter by a specific label, milestone, or assignee.

3. Use a different browser: As you noted, the board appears to be faster in Chrome than in Firefox. You may want to try using a different browser to see if that helps.

4. Contact [GitHub support](https://support.github.com/contact?tags=docs-generic): If you've tried the above steps and are still experiencing slow performance, you may want to contact GitHub support to see if there's a known issue or if they can provide additional assistance.

I hope this helps!

**Reply by Anmol-Baranwal** [reply]
@sirex 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by sirex** [reply]
@Anmol-Baranwal your answer is useful, but does not solve my issue, I use filters, tried different browsers, but issue still exists.

**Reply by Anmol-Baranwal** [reply]
No worries, let's wait for others' answer

**Comment by dcarsey** [comment]
I am also Googling about this problem.  Every action (click) on the board takes a long time to respond.  It's been this way for at least a few weeks.

**Comment by alopetegui** [comment]
I'm also having same problem since three weeks ago

**Comment by dbrasseur-aneo** [comment]
Same issue here

**Comment by dan-castillo** [comment]
same issue here I  have a basic Kanban board with about 2100 cards with and seven columns and it became very slow about a month ago - when I archived approximately 850 cards the speed came back to normal 

Moral of the story: clean up the boards - although not sure why only 2100 cards make a modern system slow - don't get me wrong I love Github I have been using it for years and this is the first time that I have even encountered any issue so no big deal if I have to archive!

**Comment by matt-resto-guru** [comment]
Guys this is insane, the frontend is so slow. 300 cards. Can't scroll or do anything, really ridiculous. Use virtualized lists

**Comment by jbergenblom** [comment]
We just started using gh projects a week ago, have around 20 cards on the board, and can barely assign a label...

**Comment by bleal-kitmanlabs** [comment]
Same issue here. Its being very annoying at some stage.

**Comment by mirtariq** [comment]
+1 - very slow to the point of being unusable

**Comment by qf-tyler-tel** [comment]
+1, very slow. Our board is 8 columns, only about 1,000 cards.
One thing to note for me - some filters actually make it slower for me. If I filter by Repo it is taking about 30-60 seconds for each click action (move card, reassign, archive etc.). Not having the filters reduces it down to about half that, still nearly unusable, but then I see a bunch of cards I don't need to see for the work I'm doing. Archiving cards helps but 1,000 cards should be cake for a modern application.

**Comment by lindacmsheard** [comment]
Was this resolved last year? Experiencing laggyness when interacting with project cards that makes it not fit for purpose. Not  related to internet speed - other sites fine. I have ~100 cards only, issue also happens in list view. (A large proportion are drafts, not issues.)
